docs/reference/deploy/helm-config-reference.rst

```diff
@@ -251,4 +251,27 @@
    namespaces. Maps to the ``resource_pools`` section from the :ref:`master configuration
    <master-config-reference>`.
 
+-  ``additional_resource_managers``: This section is a list of additional resource managers for if
+   you want to launch jobs in multiple Kubernetes clusters. See :ref:`additional_resource_managers
+   <master-config-additional-resource-managers>` for information on this feature. An example
+   configuration is provided in the ``values.yaml`` file.
+
+   -  ``resource_manager``: The resource manager configuration. See :ref:`resource_manager
+      <master-config-resource-manager>` for available configuration options.
+
```

Comment:
-  ``resource_manager``: Describes the configuration settings for the resource manager. Maps to :ref:`resource_manager
      <master-config-resource-manager>` in the master configuration.